package.json

```diff
@@ -234,6 +234,14 @@
                             "type": "number",
                             "description": "Remote debugging port for Python. Usually 5678."
                         },
+                        "vs-kubernetes.dotnet-vsdbg-path": {
+                            "type": "string",
+                            "description": "The path to the vsdbg debugger in the container (.NET)."
+                        },
+                        "vs-kubernetes.dotnet-source-file-map": {
```

Comment:
Is this also new?  Does it need to be added to the defaults or docs?

Actually... it looks like it's not used at the moment, or am I missing something?

Reply:
No, this option was added in PR #736.

Reply:
Yikes!  Thanks for catching that!